hack/generate-metallb-manifests.sh

```diff
@@ -94,6 +94,13 @@ sed -i 's/securityContext\: |-/securityContext\:/g' ${FRR_MANIFESTS_DIR}/${FRR_M
 sed -i "s/- name: '{{ if .DeployKubeRbacProxies }}'/{{ if .DeployKubeRbacProxies }}/g" ${FRR_MANIFESTS_DIR}/${FRR_MANIFESTS_FILE}
 sed -i "s/- name: '{{ end }}'/{{ end }}/g" ${FRR_MANIFESTS_DIR}/${FRR_MANIFESTS_FILE}
 
+sed -i "s/app.kubernetes.io\/component: speaker/{{.SpeakerSelector}}/g" ${FRR_MANIFESTS_DIR}/${FRR_MANIFESTS_FILE}
+sed -i "s/app.kubernetes.io\/component: controller/{{.ControllerSelector}}/g" ${FRR_MANIFESTS_DIR}/${FRR_MANIFESTS_FILE}
+sed -i "s/app.kubernetes.io\/component: speaker/{{.SpeakerSelector}}/g" ${NATIVE_MANIFESTS_DIR}/${NATIVE_MANIFESTS_FILE}
+sed -i "s/app.kubernetes.io\/component: controller/{{.ControllerSelector}}/g" ${NATIVE_MANIFESTS_DIR}/${NATIVE_MANIFESTS_FILE}
+sed -i "s/app.kubernetes.io\/component=speaker/{{.SpeakerLabelSelector}}/g" ${FRR_MANIFESTS_DIR}/${FRR_MANIFESTS_FILE}
+sed -i "s/app.kubernetes.io\/component=speaker/{{.SpeakerLabelSelector}}/g" ${NATIVE_MANIFESTS_DIR}/${NATIVE_MANIFESTS_FILE}
```

Comment:
yq instead of sed ?

Reply:
That's beyond my yq skills. Also, I already commented this once, this is becoming not maintainable.
Since we have e2e in place now, it's gonna be way better to keep our set of manifest instead of this hell.